sdk/formrecognizer/ai-form-recognizer/tests.yml

```
@@ -1,22 +1,12 @@
-# Parameterize Location to the test resources deployment script. This allows
-# specifying a different Azure region to use when manually triggering the live
-# tests pipeline. This is useful for when we want to test in different
-# environments: Prod, Canary, etc.
-parameters:
-- name: Location
-  displayName: Location
-  type: string
-  default: canadacentral
-
 trigger: none
 
 stages:
   - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       PackageName: "@azure/ai-form-recognizer"
       ServiceDirectory: formrecognizer
-      Location: "${{ parameters.Location }}"
+      SupportedClouds: 'Public,UsGov,China'
       EnvVars:
-        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+        AZURE_CLIENT_ID: $(FORMRECOGNIZER_CLIENT_ID)
+        AZURE_TENANT_ID: $(FORMRECOGNIZER_TENANT_ID)
+        AZURE_CLIENT_SECRET: $(FORMRECOGNIZER_CLIENT_SECRET)
```

Might fix some of the issues you are seeing:

```suggestion
        AZURE_CLIENT_SECRET: $(FORMRECOGNIZER_CLIENT_SECRET)
        AZURE_SUBSCRIPTION_ID: $(FORMRECOGNIZER_SUBSCRIPTION_ID)
```

---

In case you're talking about the `Confidential Client is not supported in Cross Cloud request` failure in the browser, the same test should have also failed in node, and we don't see that.

---

Ah ok. I'd still add it here for consistency since people often copy/paste these yaml files when making new ones and can be missing values.

---

Okay.